dist/vl-wizard-pane.js

```diff
@@ -122,6 +122,15 @@ export class VlWizardPane extends vlElement(HTMLElement) {
     this._wizard.callback = Promise.resolve();
   }
 
+  /**
+   * Sta navigatie naar de volgende pagina's toe.
+   */
+  enableNextPanes() {
+    const panes = this._wizard.panes;
+    const nextPanes = panes.slice(panes.indexOf(this), panes.length);
```

Comment:
Kan misschien eenvoudiger: `const nextPanes = panes.slice(panes.indexOf(this));`

Ook in disableNextPanes

Reply:
Kan inderdaad korter, maar niet iedereen kent de end default en moet dit dan potentieel gaan opzoeken om de code goed te kunnen lezen. Daarom heb ik een voorkeur om expliciet te zeggen dat de splice tot het einde loopt.